modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -281,9 +281,9 @@ will only return label `:A`, because that is the only label for which traversal
 [[access-control-limitations-db-operations]]
 == Security and count store operations
 
-The rules of a security model may impact some of the database operations.
-This means extra security checks are necessary to incur additional data accesses, especially in the case of count store operations.
-These are, however, usually very fast lookups and the difference might be noticeable.
+The rules of a security model may impact some database operations.
+This is because extra security checks are necessary to incur additional data accesses, especially in the case of count store operations.
```

Comment:
```suggestion
This is because extra security checks are necessary that incur additional data accesses, especially in the case of count store operations.
```
maybe makes it clearer?

Reply:
hmm, not sure. I think I like the `to` more.

Reply:
I don't feel very strongly about it 🤷